### PR TITLE
feat: configuration page

### PR DESCRIPTION
**Pull request by chipp972**
ref: MANAGER-12488

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/vrack-services-new` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-12488
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

- [x] BDD setup with playwright
- [x] Configuration screen + tests (not fully implemented)
- [x] Onboarding screen tests

### CODE REVIEW

File: packages/manager/apps/vrack-services/src/pages/create/ZoneFormField.tsx [hunk 1]
```diff
@@ -84,18 +83,18 @@ export const ZoneFormField: React.FC<Props> = ({
                   id={zone.name}
                   key={zone.name}
                   value={zone.name}
-                  name={inputName}
-                  checked={selectedZone === zone.name}
-                  {...(isReadOnly ? { disabled: true } : {})}
+                  name={zoneInputName}
+                  checked={selectedZone === zone.name || undefined}
+                  disabled={isReadOnly || undefined}
```
Comment (lionel95200x) [inline packages/manager/apps/vrack-services/src/pages/create/ZoneFormField.tsx:88]:
```suggestion
                  disabled={!!isReadOnly}
```

Comment (chipp972) [reply packages/manager/apps/vrack-services/src/pages/create/ZoneFormField.tsx:88]: on ODS components we have to put undefined to disable a boolean property

File: packages/manager/apps/vrack-services/src/pages/index.tsx [hunk 2]
```diff
@@ -47,10 +45,14 @@ export default function ListingPage() {
     );
   }
 
-  if (error || data?.status !== 200) {
+  if (data?.data.length === 0) {
     return <Navigate to="onboarding" />;
   }
 
+  if (error || data?.status !== 200) {
```
Comment (aboungnaseng-ovhcloud) [inline packages/manager/apps/vrack-services/src/pages/index.tsx:52]:
```suggestion
  if (error) {
```
just this condition is enough now with the Promise.reject(error) in client.ts

File: packages/manager/apps/vrack-services/src/pages/dashboard/[id]/_layout.tsx [hunk 3]
```diff
@@ -58,7 +58,7 @@ export default function DashboardPage() {
   ];
 
   if (results[0].status === 'success' && results[0]?.data?.status !== 200) {
-    return <ErrorBanner error={results[0].data} />;
+    return <ErrorPage error={results[0].data} />;
```
Comment (aboungnaseng-ovhcloud) [inline packages/manager/apps/vrack-services/src/pages/dashboard/[id]/_layout.tsx:61]: you can pass error from results const { error } = results[0]

**Review by aboungnaseng-ovhcloud — APPROVED** [review]

File: packages/manager/apps/vrack-services/src/pages/dashboard/[id]/_layout.tsx (hunk 3 above)
Comment (chipp972) [reply packages/manager/apps/vrack-services/src/pages/dashboard/[id]/_layout.tsx:61]: I will fix when I rebase on dev-transformation the main branch

File: packages/manager/apps/vrack-services/src/pages/index.tsx (hunk 2 above)
Comment (chipp972) [reply packages/manager/apps/vrack-services/src/pages/index.tsx:52]: fixed on another PR

**Pull request merged**